include/dao/dao.js

```diff
@@ -85,6 +85,9 @@ module.exports = function DAOModule(pb) {
      */
     DAO.DESC = -1;
 
+    var GLOBAL_SITE = 'global';
```

Comment:
This should probably be pulled from the configuration rather than a private variable in DAO.